src/deployment/cd.md

```diff
@@ -193,6 +193,112 @@ secrets in pull requests that you accept and merge.
          * `cd android` or `cd ios`
          * `bundle exec fastlane [name of the lane]`
 
+## Xcode Cloud
+
+[Xcode Cloud][] is a continuous integration and delivery service for building,
+testing, and distributing apps and frameworks for Apple platforms.
+
+### Requirements
+
+* Xcode 13.4.1 or higher.
+* Be enrolled in the [Apple Developer Program][].
+
+### Custom build script
+
+Xcode Cloud recognizes [custom build scripts][] that can be 
```

Comment:
I think `[custom build scripts][]` is broken now since you changed it to `[custom build script]` 🙂

Reply:
@jmagman that completely slipped me earlier